stable/elasticsearch-curator/values.yaml

```diff
@@ -100,3 +100,6 @@ priorityClassName: ""
 #   - name: es-certs
 #     mountPath: /certs
 #     readOnly: true
+
+securityContext: {}
```

Comment:
@mvisonneau wdyt to add default values here?

Reply:
Yes I don't mind, I did not want to change the default behaviour of the chart.

Reply:
done 👍